packages/extension-ui/src/components/RadioGroup.tsx

```diff
@@ -28,14 +36,55 @@ const RadioGroup: React.FC<Props> = ({ className, defaultSelectedValue, onSelect
     [onSelectionChange]
   );
 
+  console.log('options', options);
+
+  const { alephOptions, otherOptions } = options.reduce(
+    (acc: { alephOptions: Option[]; otherOptions: Option[] }, option) => {
+      const isAleph = withTestNetwork ? option.text.includes('Aleph Zero') : option.text === 'Aleph Zero';
+
+      if (isAleph) {
+        acc.alephOptions.push(option);
+      } else if (withTestNetwork || !option.text.includes('Aleph Zero')) {
+        acc.otherOptions.push(option);
+      }
+
+      return acc;
+    },
+    { alephOptions: [], otherOptions: [] }
+  );
+
+  function getPositionForOption(options: Option[], index: number): 'top' | 'bottom' | 'both' | 'middle' {
```

Comment:
I haven't dived too deep into this, but wouldn't the same logic be achievable in css-land only with `:first-of-type` & `:last-of-type` (or similar) pseudo classes?

Reply:
Nice catch! Yeah, I'll refactor it